### PR TITLE
fix lc plots to handle comets

### LINKED ISSUE #575: Comets (obviously) break the orbit plotter
see https://rollbar.com/LCO/NEOexchange/items/234/?utm_campaign=new_item_message&utm_medium=slack&utm_source=rollbar-notification

specifically, this break happens when we try to set the plot bounds using semi-major axis, but I'd bet it breaks elsewhere too...

### PR DESCRIPTION
**Pull request by jchate6**
closes #575 and rollbar issue.

this fix is ready for deploy, but it was a Friday evening, and I haven't finished tests.

Finished a test. 

This will behave as normal for parabolic comets, but will not plot an orbit for non-periodic comets because hyperbolas and parabolas make the code angry, and not even in the same way.

### CODE REVIEW

File: neoexchange/astrometrics/ephem_subs.py [hunk 1]
```diff
@@ -2247,12 +2247,17 @@ def convert_to_ecliptic(equatorial_coords, coord_date):
...
+    """Give Cartesian coordinates for a given true anomaly in radians for a given set of orbital elements"""
     a = body_elements['meandist']
     e = body_elements['eccentricity']
     i = radians(body_elements['orbinc'])
     om = radians(body_elements['longascnode'])
     w = radians(body_elements['argofperih'])
+    if not a:  # Comet
+        if e < 1:  # Periodic
+            a = body_elements['perihdist'] / (1 - e)
+        else:  # Non-periodic, not going to deal with this right now.
```
Comment (talister) [inline neoexchange/astrometrics/ephem_subs.py:2259]: Think you can use `r = q * (1+e) / (1+e*cos(nu))` to calculate radius vector for objects with _e_=0.98..1.1. Example object: http://127.0.0.1:8000/target/60979/lc/

Comment (jchate6) [reply neoexchange/astrometrics/ephem_subs.py:2259]: Technically, the given equation is accurate for everything but e=1. But ultimately it's a plotting issue I was having. 
I'm trying to resolve it, but I noticed we were getting more rollbar issues and wanted to get the PR ready.

**Comment by jchate6** [comment]
@talister I believe this version should accommodate all comets.

**Comment by talister** [comment]
> @talister I believe this version should accommodate all comets.

A bold claim... ;-)

**Review by talister — APPROVED** [review]

**Pull request merged**